js/app/cart/Cart.jsx

```diff
@@ -101,7 +101,6 @@ class Cart extends React.Component
             }
 
         $.post(this.props.addToCartURL, {
-          date: this.props.deliveryDate,
```

Comment:
Is this normal?

Reply:
Yep (I dont remember why this change is here though)

Reply:
I extracted it into a single commit 😉